src/components/app.test.js

```diff
@@ -1,26 +1,34 @@
-import React from 'react';
-import {shallow, mount} from 'enzyme';
-import App from './app';
-import BasicTutorial from './tutorials/basic-tutorial';
-import DeveloperPreview from './developer-preview';
+import { render } from 'react-dom'
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
```

Comment:
That's the react router standard for tests: https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/guides/testing.md